### PR TITLE
Image and input

### PR DESCRIPTION
**Pull request by Megha-Joshi**
Screenshot (46)
Screenshot (47)
Screenshot (48)
Screenshot (49)
Screenshot (50)

This is my image and input component. I still have to add carbon code.

### CODE REVIEW

**Review by dev-enforced — APPROVED** [review]
Code and output generated by it be it the Image and input ones looks good to me !! You can merge it once you add the carbon related code

File: css/nav.css [hunk 1]
```diff
@@ -58,12 +64,12 @@ body {
 .right-nav {
     display: flex;
     margin-left: auto;
-    flex: 0 0 300px;
-    height: 200px;
+    flex: 0 0 18.75rem;
+    height: 12.5rem;
```
Comment (adith-a-danthi) [inline css/nav.css:68]: These units could have been in `px` itself

File: Components/Image/image.html [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="\css\nav.css">
+    <link rel="stylesheet" href="\css\doc.css">
+    <link rel="stylesheet" href="\Components\Image\image.css">
+    <title>Document</title>
+</head>
+
+<body>
+    <div class="container">
```
Comment (adith-a-danthi) [inline Components/Image/image.html:18]: You can replace `div` with a semantic tag `header`

File: Components/Image/image.html [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+                <li>
+                    <a href="/">Releases</a>
+                </li>
+            </ul>
+        </nav>
+    </div>
+
+
+    <!-- Docs -->
+    <div class="main">
```
Comment (adith-a-danthi) [inline Components/Image/image.html:39]: You can use `main` tag instead of `div` and `aside` for the sidebar

**Review by adith-a-danthi — CHANGES_REQUESTED** [review]
Looks good! Try to use semantic tags `header`, `aside`, `main`.

**Review by Jia-A — APPROVED** [review]
Nice use of variables throughout the code. Good formatting. Ready to merge.

**Pull request merged**